repokid/__init__.py

```diff
@@ -42,7 +42,7 @@ def init_config() -> RepokidConfig:
         None
     """
     load_config_paths = [
-        os.path.join(os.getcwd(), "config.json"),
+        os.environ.get("REPOKID_CONFIG_LOCATION", os.path.join(os.getcwd(), "config.json")),
```

Comment:
Can you split this into two lines? (1) the env var, which will take precedence and (2) the working directory like it was before

Reply:
Looks like you also need to run a `pre-commit install` to fix the linting error in the builds